website/docs/r/application.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_application"
+sidebar_current: "docs-newrelic-resource-application"
+description: |-
+  Manage configuration for an existing application in New Relic.
+---
+
+# Resource: newrelic_application
+
+Use this resource to manage configuration for an application that already exists in New Relic.
```

Comment:
Should make a larger call-out that you can not Create a new application this way.

Reply:
Could use one of the markdown callouts that TF supports...

`~>` for a beige background callout
`->` for a blue background callout 

There's also one for green, but can't remember what it is off the top of head.

Reply:
Updated to include a brighter note.